src/Config.ts

```diff
@@ -1,15 +1,18 @@
 import * as path from 'path';
 import * as fs from 'fs-extra';
+const Web3 = require('web3');
 import { EncryptedKeystoreV3Json } from 'web3-eth-accounts';
 import { Validator as JsonSchemaVerifier } from 'jsonschema';
-import { MosaicConfig } from './MosaicConfig';
+import MosaicConfig from './MosaicConfig';
 import Directory from './Directory';
-import InvalidFacilitatorConfigException from './Exception';
+import InvalidFacilitatorConfigException, { WorkerPasswordNotFoundException } from './Exception';
 import * as schema from './Config/FacilitatorConfig.schema.json';
 import Utils from './Utils';
+import Account from './Account';
 
 // Database password key to read from env.
 const ENV_DB_PASSWORD = 'MOSAIC_FACILITATOR_DB_PASSWORD';
+export const ENV_WORKER_PASSWORD_PREFIX = 'MOSAIC_ADDRESS_PASSW_';
```

Comment:
Do we need to export this const?

Reply:
We use this variable in some of our test cases thus added export here.